torrt/compat.py

```diff
@@ -34,12 +34,12 @@ def base64encode(string_or_bytes):
         """Return base64 encoded input
 
         :param string_or_bytes:
-        :return: str
+        :return: bytes
```

Comment:
Hi,
Thank you. 

Python 2 support will be dropped early 2020, but still.
Now that ``base64encode`` in Py2 is said to returns bytes please update `:return:` for Py2 acordingly. Then we're ready to merge.